### PR TITLE
2970 add card to settings

### PR DESCRIPTION
**Pull request by AJAYSINGHBISHT12**
Targets #2970
### Description
Added a card in the setting file
### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -117,6 +117,9 @@
         Route::get('/nda-template', 'NDAAgreementController@index')->name('setting.agreement.nda');
         Route::get('/invoice-template', 'SettingController@invoiceTemplates')->name('setting.invoice');
         Route::post('/invoice-template', 'SettingController@updateInvoiceTemplates')->name('setting.invoice.update');
+        Route::prefix('bankdetails')->group(function () {
```
[the remark this answers is not in the document]
Comment (GauravGusain98) [reply routes/web.php:120]: These are 2 different words

File: app/Http/Controllers/Settings/bankController.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+namespace App\Http\Controllers\Settings;
+
+use App\Http\Controllers\Controller;
+
+class bankController extends Controller
+{
+    public function index()
+    {
+        return view('Settings.bankdetails');
```
Comment (GauravGusain98) [inline app/Http/Controllers/Settings/bankController.php:11]: CI Check if failing. Also the naing convention of blade file is incorrect. Please check how we pass blade file name here.

File: resources/views/settings/index.blade.php
Comment (GauravGusain98) [inline resources/views/settings/index.blade.php:1]: Please correct the file naming convention

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comments

**Review by Hemrajbharti123 — APPROVED** [review]
LGTM

**Review by GauravGusain98 — APPROVED** [review]

**Pull request merged**